copy/src/App.tsx

```diff
@@ -0,0 +1,124 @@
+import React from "react";
+// import loader from './images/loader.svg'
+// import { useTheme } from "styled-components";
+import { fetchQuizQuestions, Difficulty } from "./API";
+//Styles
+import { GlobalStyle, Wrapper } from "./App.styles";
+//Components
+import QuestionCard from "./components/QuestionCard";
+//Redux
+import { useDispatch, useSelector } from "react-redux";
+import { AppState } from "./redux/reducers";
+
+import {
+  setCurrentNumber,
+  setGameOver,
+  setLoading,
+  setQuestions,
+  setScore,
+  setUserAnswers,
+} from "./redux/actions";
+
+export type AnswerObject = {
+  question: string;
+  userAnswer: string;
+  isCorrect: boolean;
+  correctAnswer: string;
+};
+
+const TOTAL_QUESTIONS = 10;
+
+function App() {
+  const isLoading = useSelector((state: AppState) => state.game.loading);
+  const questions = useSelector((state: AppState) => state.game.questions);
+  const userAnswers = useSelector((state: AppState) => state.game.userAnswers);
+  const score = useSelector((state: AppState) => state.game.score);
+  const currentNumber = useSelector(
+    (state: AppState) => state.game.currentNumber
+  );
+  const isGameOver = useSelector((state: AppState) => state.game.gameOver);
+  const dispatch = useDispatch();
+
+  const startQuiz = async () => {
+    dispatch(setLoading(true));
+    dispatch(setGameOver(false));
+
+    const newQuestions = await fetchQuizQuestions(
+      TOTAL_QUESTIONS,
+      Difficulty.Easy
+    );
+
+    dispatch(setQuestions(newQuestions));
+    dispatch(setScore(0));
+    dispatch(setUserAnswers([]));
```

Comment:
these 4 actions probably would be better consolidated into one, eg something like "startNewGame(questions)", which set all the stuff that's relevant to new game